views/update/review-update-statement.html

```diff
@@ -315,23 +314,21 @@ <h2 class="govuk-heading-l">Managing officers</h2>
     {% endfor %}
   {% endif %}
 
-  {% if addTrustsEnabled and manageTrustsEnabled %}
+  {% if addTrustsEnabled %}
     {% if hasExistingTrusts %}
       <br>
       <h2 class="govuk-heading-l">Trusts</h2>
       {% for trust in update.review_trusts %}
         {% set trust_heading = trust.trust_name %}
         {% include "includes/check-your-answers/trusts.html" %}
       {% endfor %}
+    {% elif pageParams.hasAnyBosWithTrusteeNocs %}
+      <br>
+      <h2 class="govuk-heading-l">Trusts</h2>
+      <p class="govuk-body govuk-!-margin-bottom-9">Trust information cannot be displayed.</p>
```

Comment:
I think this Trust information cannot be displayed option can be removed as it shouldn't be reachable now we have effectively set manageTrustsEnabled = true.
In previous version it was only reachable when:
    addTrustsEnabled = true
    manageTrustsEnabled = false
    pageParams.hasAnyBosWithTrusteeNocs = true

In this new version it has changed and is now reachable when :
    addTrustsEnabled = true
    hasExistingTrusts = false
    pageParams.hasAnyBosWithTrusteeNocs = true

Reply:
Not sure this makes sense as the elif is not directly related to the managed trusts if but the nested if. Not sure this made sense in the original unchanged version tbh, so removed it.

Reply:
Can have a call to discuss it, but the way to look at it is that we are now making manageTrustsEnabled permanently true, and when you look at the previous if /elif logic, when manageTrustsEnabled is true, that 'missing trusts' text can never be reached.

Reply:
This is clear now, the elif relates to managed trusts and was reachable only when the value was false, which will no longer be the case.